functests/performance/updating_profile.go

```diff
@@ -204,43 +200,60 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 			By("Waiting when MCP finishes updates and verifying new node has updated configuration")
 			waitForMcpCondition(newRole, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue, mcpUpdateTimeout)
 
-			_, err = nodes.ExecCommandOnMachineConfigDaemon(testclient.Client, newWorkerNode, []string{"ls", "/rootfs/" + testutils.PerfRtKernelPrebootTuningScript})
+			_, err = nodes.ExecCommandOnMachineConfigDaemon(testclient.Client, newCnfNode, []string{"ls", "/rootfs/" + testutils.PerfRtKernelPrebootTuningScript})
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("cannot find the file %s", testutils.PerfRtKernelPrebootTuningScript))
-			Expect(execCommandOnWorker(chkKubeletConfig, newWorkerNode)).To(ContainSubstring("topologyManagerPolicy"))
-			Expect(execCommandOnWorker(chkCmdLine, newWorkerNode)).To(ContainSubstring("tuned.non_isolcpus"))
+			Expect(execCommandOnWorker(chkKubeletConfig, newCnfNode)).To(ContainSubstring("topologyManagerPolicy"))
+			Expect(execCommandOnWorker(chkCmdLine, newCnfNode)).To(ContainSubstring("tuned.non_isolcpus"))
```

Comment:
We can check it, but in this case we need to be sure we should have it in any scenario. I tried to avoid verifications that depend on profile settings (customer can have RTKernel=False), I wanted to check only 100% existing parameters...

Reply:
ok, makes sense :+1: